automation/project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=1.9.2
```

Comment:
two questions ... 
- is this needed?
- and what if this ends up differing from the parent project build.properties?

Reply:
Oh, let me look into this. I think there was something weird about one of the SSH packages I needed where it needed sbt pinned to a lower version than the outer module, but I may have sleepily tried to execute the automation tests from the wrong directory and added this erroneously....